src/app/InteractionModelEngine.cpp

```diff
@@ -515,59 +503,31 @@ bool InteractionModelEngine::InActiveReadClientList(ReadClient * apReadClient)
 
 void InteractionModelEngine::ReleaseClusterInfoList(ClusterInfo *& aClusterInfo)
 {
-    ClusterInfo * lastClusterInfo = aClusterInfo;
-    if (lastClusterInfo == nullptr)
+    ClusterInfo * runner = aClusterInfo;
```

Comment:
```suggestion
    ClusterInfo * current = aClusterInfo;
```
might be clearer, combined with "next" below.

Reply:
updated with current, thanks, seems not combine with next?

Reply:
I meant that "current" and "next" go together in terms of terminology.